backend/corpora/lambdas/api/v1/collection.py

```diff
@@ -63,10 +82,12 @@ def get_collection(db_session, collection_uuid, visibility, **kwargs):
 
 
 @dbconnect
-def get_collection_details(collection_uuid: str, visibility: str, user: str):
+def get_collection_details(collection_uuid: str, user: str):
     db_session = g.db_session
-    collection = get_collection(db_session, collection_uuid, visibility, include_tombstones=True)
-    if collection.tombstone and visibility == CollectionVisibility.PUBLIC.name:
+    collection = Collection.get_collection(db_session, collection_uuid, include_tombstones=True)
+    if not collection:
+        raise ForbiddenHTTPException()
```

Comment:
No, because a user should only be told 1) yes, this is a collection you are authorized to view, or 2) you are not authorized to view this, and there may or may not be a collection with that uuid